content/rs/administering/cluster-operations/updating-certificates.md

```diff
@@ -5,51 +5,83 @@ weight: $weight
+- Connections between clients and database endpoints
 - Synchronization between databases for ReplicaOf and CRDB
+- Discovery Services
 
-These self-signed certificates are generated on the first node of each RS installation. These certificates are then copied to all other nodes added to the cluster.
+These self-signed certificates are generated on the first node of each RS installation and are copied to all other nodes added to the cluster.
 
-{{% note %}}When using the default self-signed certificates, an untrusted
-connection notification will appear in the management UI. If you do not
-update the self-signed certificate with your own certificate, depending
-on the browser you use, you might be able to allow the connection for
-this specific session, or add an exception to make this site trusted in
-future sessions.{{% /note %}}
+When you use the default self-signed certificates, an untrusted connection notification is shown in the web UI.
+Depending on the browser you use, you can allow the connection for each session or add an exception to make the site trusted in future sessions.
 
-## How to update SSL/TLS certificates
+{{% warning %}}
+When you update the certificates, the new certificate replaces the same certificates on all nodes in the cluster.
+{{% /warning %}}
 
-**For versions 5.2.0 and above:** ([Procedures for previous releases]({{< relref "/rs/references/procedures-previous-releases.md#Updating-SSL/TLS-certificates-for-Previous-Releases" >}}))
+## How to update TLS certificates
 
-{{% warning %}}The new certificate replaces the equivalent certificate on all nodes in the cluster. Existing certificates are overwritten.{{% /warning %}}
+The certificates that you can replace are:
 
-- Use the REST API to replace the certificate:
+- API - The certificate used by the API
+- CM - The certificate used by the management user interface or the cluster manager and the Sentinel discovery service
+- Proxy - The certificate used to establish communication between a client and the databases
+- Syncer - The certificate used to encrypt replication traffic between clusters
+- Metrics exporter - The certificate used by Prometheus
 
-    ```src
+### Using the CLI
 
-    curl -k -X PUT -u "<username>:<password>" -H "Content-Type: application/json" -d '{ "name": "<cert_name>", "key": "<key>", "certificate": "<cert>" }' https://<cluster_address>:9443/v1/cluster/update_cert
+To replace certificates using the rladmin CLI, run:
 
-    ```
+```src
+ rladmin cluster certificate set <cert-name> certificate_file <cert-file-name>.pem key_file <key-file-name>.pem
+```
+
+Where:
+
+- cert-type - The type of certificate you want to replace:
+    - For management UI: `cm`
+    - For REST API: `api`
+    - For database endpoint: `proxy`
+    - For syncer: `syncer`
+    - For metrics exporter: `metrics_exporter`
+- cert-file-name - The name of your certificate file
+- key-file-name - The name of your non-password protected key file
```

Comment:
Why do we need to say "non-password protected"?

Reply:
@IAmATeaPot418 What's the significance that it is non-password protected?